### DISCUSSION TITLE
Making it work without Open AI

### DISCUSSION THREAD

**Discussion by p-christ** [567-labs/instructor#364] [Q&A] [answered]
Is there anyway this can be made to work with ordinary open source models privately hosted?

**Comment by j4k0bk** [comment]
Bump!

**Comment by jxnl** [comment, marked as answer]
https://github.com/jxnl/instructor/tree/main/examples/ollama

**Reply by jxnl** [reply]
https://jxnl.github.io/instructor/examples/ollama/
